models/tasks.js

```diff
@@ -174,9 +191,32 @@ const fetchUserActiveAndBlockedTasks = async (username) => {
  */
 
 const fetchUserCompletedTasks = async (username) => {
-  return await fetchUserTasks(username, ['completed'])
+  return await fetchUserTasks(username, ['completed', 'COMPLETED'])
 }
 
+/**
+ * Fetch all overdue tasks
+ * @param overdueTasks <Array>: tasks which are overdue
+ * @return {Promsie<Array>}
+ */
+const overdueTasks = async (overDueTasks) => {
+  try {
+    const newAvailableTasks = await Promise.all(overDueTasks.map(async (task) => {
+      let { status, assignee, id } = task
+      status = TASK_STATUS.UNASSIGNED
```

Comment:
Since we are moving with a new task workflow let's make this AVAILABLE

Reply:
done.